site/topics/program-counter/esap_alu_ram_output_pc.dig

```diff
@@ -14,7 +14,7 @@
     </entry>
```

Comment:
Swapped out module and reordered control signals
![image](https://github.com/user-attachments/assets/55770a6b-058b-452d-8235-e70df2b9c941)